<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkManagement</class>
 <widget class="QDialog" name="networkManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffe</string>
   </property>
   <property name="title">
    <string>Server Address</string>
   </property>
   <widget class="QLineEdit" name="serverAddress">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectionTest">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>90</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Connection</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="systemCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#fef</string>
   </property>
   <property name="title">
    <string>Program ID</string>
   </property>
   <widget class="QLineEdit" name="activationCode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Activation Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="validateCode">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
